server/app/models/Application.java

```diff
@@ -34,6 +35,7 @@ public class Application extends BaseModel {
   @ManyToOne private Program program;
 
   @OneToMany(mappedBy = "application")
+  @OrderBy("createTime desc")
```

Comment:
Not blocking: Would it make sense to add an index on createTime?

Reply:
I hadn't thought about it either way.  Given we don't search on this and I don't expect there to be many events per application I lean towards an index being wasteful than much benefit.

thoughts?

Reply:
I'm indifferent. Just thought I'd point it out here as an area for optimization.